certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
-            "description": "OK",
+            "description": "Success",
             "schema": {
-              "type": "string",
-              "x-ms-summary": "JSON result",
-              "description": "Result in JSON"
+              "$ref": "#/definitions/FileResponse"
             }
           }
         },
-        "summary": "Convert XML to JSON",
-        "operationId": "ConvertXmlToJson",
+        "x-ms-visibility": "important",
+        "summary": "Add Table to a Word (DOCX) file",
+        "description": "Add Table to a Word (DOCX) file or create new Word (DOCX) file with image"
+      }
+    },
+    "/AddTextToWord": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "AddTextToWord",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
         "parameters": [
           {
-            "name": "XML",
             "in": "body",
-            "required": true,
+            "name": "wordText",
             "schema": {
-              "type": "string",
-              "x-ms-visibility": "important"
-            },
-            "x-ms-visibility": "important"
+              "$ref": "#/definitions/WordText"
+            }
           }
         ],
-        "description": "Convert XML to JSON",
-        "x-ms-visibility": "important"
-      }
-    },
-    "/ConvertSharePointSearchResults": {
-      "post": {
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success",
             "schema": {
-              "type": "array",
-              "items": {
-                "type": "object",
-                "properties": {
-                  "title": {
-                    "type": "string",
-                    "description": "Title of SharePoint ressource",
-                    "title": "title"
-                  },
-                  "link": {
-                    "type": "string",
-                    "description": "Link of SharePoint ressource",
-                    "title": "link"
-                  }
-                }
-              }
+              "$ref": "#/definitions/FileResponse"
             }
           }
         },
-        "summary": "Convert SharePoint search results to list",
-        "description": "Convert SharePoint search results to a simple list",
-        "operationId": "ConvertSharePointSearchResults",
         "x-ms-visibility": "important",
-        "parameters": [
-          {
-            "name": "Content-Type",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "default": "application/json",
-            "x-ms-visibility": "internal",
-            "description": "JSON format",
-            "x-ms-summary": "JSON format"
-          },
-          {
-            "name": "Result",
```

Comment:
@stsiek The type has been changed for the input field. Existing flows will continue to send the field with the old type.